components/main.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { useContext, useState } from "react";
+import dynamic from "next/dynamic";
+
+import CovidBanner from "@components/banners/covid";
+import HoursBanner from "@components/banners/hours";
+import HelpToggle from "@components/buttons/help";
+import LocationListToggle from "@components/buttons/location";
+import Help from "@components/help";
+import LocationList from "@components/locationList";
+import SEO from "@components/seo";
+import Welcome from "@components/welcome";
+
+import useLocation from "@hooks/useLocation";
+
+const Mapbox = dynamic(() => import("@components/mapbox"), {
```

Comment:
What does `dynamic` do?

Reply:
This is a [dynamic import](https://nextjs.org/docs/advanced-features/dynamic-import)! Since mapbox is only a client-side JS library, we're only going to import it in the client-side bundle. This also allows us to add this little "Loading Mutual Aid Sites..." loading state before the map renders!